.github/workflows/android_build_and_test.yml

```diff
@@ -2,19 +2,17 @@ name: android_build_and_test
 
 on:
   push:
-    branches: [ main ]
```

Comment:
I suggest separating the iOS and Android PRs. The build/test process with RN is so flaky that it is likely much harder to try and land both platforms at once.

Reply:
Fair.  Done.